File: BlockImages.js
(async function() {
  const chrome = require('selenium-webdriver/chrome');
  const options = new chrome.Options();
  options.setUserPreferences({
    'profile.managed_default_content_settings.images': 2
  });
  const driver = chrome.Driver.createSession(options);
  await driver.get('https://selenium.dev/');
  await new Promise(resolve => setTimeout(resolve, 2000));
  await driver.quit();
})();

File: BlockImages.py
from selenium import webdriver
from time import sleep

opt = webdriver.ChromeOptions()
opt.add_experimental_option('prefs', {
    'profile.managed_default_content_settings.images': 2
})
driver = webdriver.Chrome(options=opt)
driver.get('https://selenium.dev/')

sleep(2)
driver.quit()
